src/Microsoft.ML.DnnImageFeaturizer.ResNet50/Microsoft.ML.DnnImageFeaturizer.ResNet50.csproj

```diff
@@ -0,0 +1,50 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <UsingTask TaskName="DownloadFilesFromUrl" AssemblyFile="$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll"/>
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeInPackage>Microsoft.ML.DnnImageFeaturizer.ResNet50</IncludeInPackage>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.ML.OnnxTransform\Microsoft.ML.OnnxTransform.csproj" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <LocalAppDataDir>$(LocalAppData)</LocalAppDataDir>
```

Comment:
The build should NOT be doing anything with `$(LocalAppData)` on a developers machine. That is a recipe for issues on the build servers.

Reply:
Instead, these files should be downloaded to somewhere under the `$RepoRoot\bin\obj` directory.

---
In reply to: [232771603](https://github.com/dotnet/machinelearning/pull/1447#discussion_r232771603) [](ancestors = 232771603)